src/main/java/org/spongepowered/common/interfaces/entity/player/IMixinEntityPlayerMP.java

```diff
@@ -60,4 +64,9 @@ default boolean usesCustomClient() {
     void setPacketItem(ItemStack itemstack);
 
     void refreshExp();
+
+    Optional<WorldBorder> getWorldBorder();
```

Comment:
You can access this method using the `Player` interface, there is no need to have it here.